src/Type/Contacts.php

```diff
@@ -7,36 +7,50 @@
 class Contacts
 {
     /**
-     * @var \OpenEuropa\EPoetry\Type\ContactPerson[]
+     * @var null|\OpenEuropa\EPoetry\Type\ContactPerson[]
      */
-    protected $contact = [];
+    protected $contact;
 
     /**
-     * @param \OpenEuropa\EPoetry\Type\ContactPerson $contact
+     * @param ContactPerson ...$contacts
      *
      * @return $this
      */
-    public function addContact($contact): Contacts
+    public function addContact(...$contacts): Contacts
     {
-        $this->contact[] = $contact;
+        foreach ($contacts as $contact) {
+            $this->contact[] = $contact;
```

Comment:
`$this->contact` may be null here.
I would prefer not using a `foreach` and instead a
`$this->contact = array_merge($this->contact, $contacts)`
... but for this `$contact` would need to be initialized in the beggining.

Reply:
Fixed.